test/Sema/placeholder_type.swift

```diff
@@ -110,13 +110,13 @@ extension Bar {
 }
 
 // FIXME: We should probably have better diagnostics for these situations--the user probably meant to use implicit member syntax
-let _: Int = _() // expected-error {{type placeholder not allowed here}} expected-error {{type of expression is ambiguous without more context}}
-let _: () -> Int = { _() } // expected-error 2 {{type placeholder not allowed here}} expected-error {{unable to infer closure type in the current context}}
+let _: Int = _() // expected-error {{type placeholder not allowed here}} expected-error {{could not infer type for placeholder}}
```

Comment:
A request, could be done separately - we need to turn `type placeholder not allowed here` error into a fix and check whether it exists before recording `specify type for placeholder` because we'd only want to produce "not allowed here" error and not both.

Reply:
Let’s do this in a follow-up PR.

Reply:
Sounds good!